### DISCUSSION TITLE
Project set-up on Mac-OS operating system

### DISCUSSION THREAD

**Discussion by karthiknadar1204** [Hyperfoil/Horreum#391] [General]
I have installed all the utilities and packages required to set up the project on the local system, tried pasting the terminal commands but still no result
<img width="644" alt="Screenshot 2023-02-16 at 9 09 24 PM" src="https://user-images.githubusercontent.com/95973644/219415318-6932bd92-dfa3-4ec2-942a-f5ac93ffd42f.png">

Please help me with it.

**Comment by jesperpedersen** [comment]
We are looking to get rid of the `infra/` directory - as using Windows/MacOS is an issue atm.

Hopefully it will be sorted soon'ish.

You will have to use a Linux based platform for now for development

**Comment by karthiknadar1204** [comment]
which linux type and which version should i use?

**Reply by jesperpedersen** [reply]
Any distro should work - you just need docker or podman installed. I use Fedora... You should be able to run it in a VM as well.
